src/concrete/ml/onnx/convert.py

```diff
@@ -251,23 +270,39 @@ def get_equivalent_numpy_forward_from_onnx(
             Defaults to True.
 
     Returns:
-        Callable[..., Tuple[numpy.ndarray, ...]]: The function that will execute
-            the equivalent numpy function.
+        ONNXAndNumpyForwards: The function that will execute the equivalent numpy function.
     """
 
-    equivalent_onnx_model = preprocess_onnx_model(onnx_model, check_model)
+    onnx_preprocessing, equivalent_onnx_model = preprocess_onnx_model(onnx_model, check_model)
```

Comment:
I don't get why we need both onnx models here and in the following

Reply:
Because we need the onnx_preprocessing for the pre-processing later. On the client machine and for any kind of forward of the quantized_module

Reply:
yeah make sense I guess